project/SparkBuild.scala

```diff
@@ -765,16 +750,6 @@ object CopyDependencies {
 
 }
 
-object Java8TestSettings {
-  import BuildCommons._
-
-  lazy val settings = Seq(
-    javacJVMVersion := "1.8",
-    // Targeting Java 8 bytecode is only supported in Scala 2.11.4 and higher:
-    scalacJVMVersion := (if (System.getProperty("scala-2.10") == "true") "1.7" else "1.8")
```

Comment:
Hm, I can't tell from the latest master-compile-2.10 output: https://amplab.cs.berkeley.edu/jenkins/view/Spark%20QA%20Test%20(Dashboard)/job/spark-master-compile-maven-scala-2.10/3678/consoleFull
https://amplab.cs.berkeley.edu/jenkins/view/Spark%20QA%20Test%20(Dashboard)/job/spark-master-compile-maven-scala-2.10/configure

Pardon @shaneknapp but would you be able to answer whether the Spark Scala 2.10 builds actually use a Scala 2.10 binary, or something later?

If it's actually using 2.10, I suppose we could just decide to run all builds with at least Scala 2.11 even those cross-compiled for 2.10.